ui/apps/platform/src/routePaths.js

```diff
@@ -42,6 +42,8 @@ export const userRolePath = `${userBasePath}/roles/:roleName`;
 export const systemConfigPath = `${mainPath}/systemconfig`;
 export const complianceBasePath = `${mainPath}/compliance`;
 export const compliancePath = `${mainPath}/:context(compliance)`;
+export const complianceEnhancedBasePath = `${mainPath}/compliance-enhanced`;
+export const complianceEnhancedPath = `${mainPath}/:context(compliance-enhanced)`;
```

Comment:
Let’s brainstorm with the UI team on the address. Until now, we have dodged the problem:
* Replace classic /main/access with /main/access-control
* Replace classic /main/network with /main/network-graph (although interesting discussion today that network might expand beyond the graph)
* Replace classic /main/vulnerability-management with /main/vulnerabilities (this is the closest precedent, because both will exist side-by-side for similar time as compliance)

Reply:
Yeah I don't like having to use compliance-enhanced as the route but I also couldn't come up with a better alternative. A team discussion would probably be good.

Reply:
We should consider dropping use of `:context`, because that was built for the Compliance/Config Management/Vuln Management "workflow".